spec/controllers/lambda_callback/address_proof_result_controller_spec.rb

```diff
@@ -12,19 +12,19 @@
       it 'accepts and stores successful address proofing results' do
         applicant = { phone: Faker::PhoneNumber.cell_phone }
         document_capture_session.store_proofing_pii_from_doc(applicant)
-        proofer_result = AddressMock.new.proof(applicant)
+        proofer_result = Idv::Agent.new(applicant).proof_address(document_capture_session)
 
         post :create, params: { result_id: document_capture_session.result_id,
                                 address_result: proofer_result.to_h }
 
         proofing_result = document_capture_session.load_proofing_result
-        expect(proofing_result.result).to eq({ exception: '', success: 'true' })
+        expect(proofing_result.result).to include({ exception: '', success: 'true' })
       end
 
       it 'accepts and stores unsuccessful address proofing results' do
         applicant = { phone: '7035555555' }
```

Comment:
should we replace this hardcoded phone # with our new constants from the gem?

Reply:
oh yep, went through and replaced any use of them with the constant